src/components/accessories/profilePicture/ProfilePicture.tsx

```diff
@@ -1,25 +1,31 @@
-import React, { Fragment, FunctionComponent, useRef, useState } from "react";
+import React, { Fragment, FunctionComponent, useRef, useState, useEffect } from "react";
 import profilePicturePlaceholder from "../../../assets/profilePicturePlaceholder.png";
 import "./styles.scss";
-import { handlePictureSelection } from "./utils";
+import { handlePictureSelection, preprocessImage } from "./utils";
 import EditRoundedIcon from "@material-ui/icons/EditRounded";
 import DeleteRoundedIcon from "@material-ui/icons/DeleteRounded";
 
 export interface IProps {
   isEditable: boolean;
-  preLoadedBlobPic?: string;
+  preLoadedPicture?: string;
 }
 
 export const ProfilePicture: FunctionComponent<IProps> = ({
   isEditable,
-  //   preLoadedBlobPic,
+  preLoadedPicture,
 }) => {
-  //TODO: Resolve preloaded picture and set it as default
+  
   const [pic, setPic] = useState({
     preview: profilePicturePlaceholder,
     blob: "",
   });
 
+  useEffect(() => {
+    if(preLoadedPicture) {
+      preprocessImage(setPic, preLoadedPicture);
+    }
+  }, []);
```

Comment:
The React `useEffect` documentation says that we have to add to the dependency array any piece of state or prop that is being used inside the effect function. In this case, we are using `preLoadedPicture`, which is a prop. So the hook should be written as I suggest here.
```suggestion
  useEffect(() => {
    if(preLoadedPicture) {
      preprocessImage(setPic, preLoadedPicture);
    }
  }, [preLoadedPicture]);
```